docs/installation.rst

```diff
@@ -23,26 +23,27 @@ To install Pulp Smash from source (`GitHub`_):
 
     git clone https://github.com/PulpQE/pulp-smash.git
     cd pulp-smash
-    python setup.py install
+    pip install .
 
-To install Pulp Smash from source in "develop mode," where changes to source
-files are reflected in the working environment:
+Pulp Smash can also be installed from source in "develop mode," where changes to
+source code are reflected in the working environment:
 
 .. code-block:: sh
 
     git clone https://github.com/PulpQE/pulp-smash.git
     cd pulp-smash
     pip install -r requirements.txt -r requirements-dev.txt
+    pip install -e .
```

Comment:
`pip install -r requirements.txt` already does `-e .` do we still want this extra line?